frodo.js

```diff
@@ -1,48 +1,47 @@
-const objectValues = Object.values(object);
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.